### PR TITLE
Added info about transifex and translations/localizations

### PR DESCRIPTION
**Pull request by bsclifton**
Added info about transifex and translations/localizations
- new translations.md under docs/
- link is now in README.md

Auditors: @aekeus, @luixxiul

### CODE REVIEW

**Comment by bsclifton** [comment]
quick heads up - I had accidentally had assigned you both to this https://github.com/brave/browser-laptop/issues/5139; if you got an alert or email just ignore it :smile:

File: docs/translations.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# How to contribute translations
+
+There are several things you can do to help us internationalize Brave and provide a great experience for everybody
+
+## Providing a translation
```
Comment (aekeus) [inline docs/translations.md]: We generally pull in all languages files at the time we cut a release. That allows us to keep everything up to date in a scalable way.

**Comment by luixxiul** [comment]
We should localize this info itself later ;-)

File: docs/translations.md (hunk 1 above)
Comment (bsclifton) [inline docs/translations.md]: Added this info in, thanks @aekeus :smile:

**Pull request merged**